Open Judge System/Web/OJS.Web/App_GlobalResources/Areas/Contests/Shared/ContestsAllContestSubmissionsByUser.resx

```diff
@@ -123,6 +123,9 @@
   <data name="Compile_time_error" xml:space="preserve">
     <value>Compile time error</value>
   </data>
+  <data name="Deleted_submissions_info_message" xml:space="preserve">
+    <value>Submissions older than {0} year, except for the best ones per problem are deleted. All submissions older than {1} years are also deleted.</value>
```

Comment:
"{placeholder} year" and "{placeholder} years"? I'm not sure which one is correct, but you probably want to be consistent there.